modules/openxr/scene/openxr_hand.cpp

```diff
@@ -160,30 +188,102 @@ void OpenXRHand::_get_bones() {
 		"Little_Tip",
 	};
 
+	// Godot Humanoid skeleton rig bone names ("Left" or "Right" will be prepended)
```

Comment:
```suggestion
	// The Godot Humanoid skeleton rig bone names ("Left" or "Right" will be prepended).
```

We use sentence case in comments in Godot Engine.

Reply:
Fixed the new comments to be full sentences.